spec/models/customer_spec.rb

```diff
@@ -1,8 +1,85 @@
 require 'rails_helper'
 
 RSpec.describe Customer, type: :model do
+
   describe 'relationships' do
     it { should have_many :invoices }
+    # it { should have_many :transactions, through: :invoices}
+  end
+
+  describe 'validations' do
+    it {should validate_presence_of :first_name}
+    it {should validate_presence_of :last_name}
   end
 
+  before(:each) do
```

Comment:
This is awesome! Rich or I could probably clean this up with the Faker Gem and might be good to learn how to use to speed things up moving forward.